### PR TITLE
updated hammerhead, bumped testcafe version (close #1912, #1922)

### LINKED ISSUE #1912: Angular app without "#" in url can not be accessed
### Are you requesting a feature or reporting a bug?
- reporting a bug

### What is the current behavior?
- Angular app without "#" in url can not be accessed 

### What is the expected behavior?
- Angular app without "#" in url can be accessed 

### How would you reproduce the current behavior (if this is a bug)?
- Try to access the following url with your test script:  https://manage-test.adtechmedia.io/accounts/signup 

#### Provide the test code and the tested page URL (if applicable)
Tested page URL:
- https://manage-test.adtechmedia.io/accounts/signup 
Test code
```
import { Selector } from 'testcafe';

fixture `Angular TodoMVC`
  .page('https://manage-test.adtechmedia.io/accounts/signup');
 
test('Check the application title', async t => {
  const appTitle = Selector('h1');

  await t.expect(appTitle.textContent).eql('todos');
}); 
```
### Specify your

* operating system: macOS Sierra version 10.12.6
* testcafe version: 0.18.1
* node.js version: v8.5.0

### PR DESCRIPTION
**Pull request by miherlosev**
@AlexanderMoskovkin

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+### Enhancements
+
+#### :gear: WebSockets support ([testcafe-hammerhead/#911](https://github.com/DevExpress/testcafe-hammerhead/issues/911))
+TestCafe now provides full-featured WebSockets support (`wss` and `ws` protocols, request authentication, etc.). 
+
+### Bug Fixes
+* TestCafe can now click on elements that are located under the Status bar and have the `transition` css property specified ([#1934](https://github.com/DevExpress/testcafe/issues/1934))
+* Added support for pages with the `rest` and `default parameter` instructions ([testcafe-hammerhead/#1336](https://github.com/DevExpress/testcafe-hammerhead/issues/1336))
+* Pages with several `base` tags are supported now ([testcafe-hammerhead/#1349](https://github.com/DevExpress/testcafe-hammerhead/issues/1349))
+* Redirects from cross-domain to same-domain pages are processed now ([testcafe-hammerhead/#1363](https://github.com/DevExpress/testcafe-hammerhead/issues/1363))
```
Comment (AlexanderMoskovkin) [inline CHANGELOG.md]: This issue is still opened in the repository. Should we close it or remove it from the changelog?
/cc @miherlosev @LavrovArtem

Comment (miherlosev) [reply CHANGELOG.md]: We've are fixed the partial case. The problem covers whole url processing mechanism.
This is why we don't close an appropriate issue.

**Pull request merged**